test/sharness/t0100-name.sh

```diff
@@ -10,12 +10,14 @@ test_description="Test ipfs repo operations"
 
 test_init_ipfs
 
+for TTL_PARAMS in "" "--ttl=0s" "--ttl=1h"; do
```

Comment:
wow the rest of the file is in a for loop without indentation. didnt even catch this the first time. please dont do that. 

do this instead:

```sh
somefunc() {
  ttl=$1
  all the tests here
}

for TTL in "" "--ttl=0s" "--ttl=1h"; do
  somefunc $TTL
done
```